db/seeds.rb

```diff
@@ -59,101 +59,18 @@ def seed_nonprofit_applications
-  Member.find_or_create_by! first_name: 'Max', last_name: 'Wolffe',
-                            email: 'max.alan.wolffe@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Kevin', last_name: 'Shieh',
-                            email: 'kevshieh@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Warren', last_name: 'Shen',
-                            email: 'warrenzshen@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Will', last_name: 'Tang',
-                            email: 'williamctang93@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Aleks', last_name: 'Kamko',
-                            email: 'aykamko@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Varun', last_name: 'Rau',
-                            email: 'varun@berkeley.edu',
-                            member_role: leader_role
-  Member.find_or_create_by! first_name: 'Nishant', last_name: 'Desai',
-                            email: 'nish.sdesai@gmail.com',
-                            member_role: leader_role
-  Member.find_or_create_by! first_name: 'Jordeen', last_name: 'Chang',
-                            email: 'jordeenchang@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Sameera', last_name: 'Vemulapalli',
-                            email: 'sameera.vemulapalli@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Ethan', last_name: 'Chiou',
-                            email: 'nahte.uoihc@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Atsunori', last_name: 'Kakitani',
-                            email: 'atsunori.kakitani@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Elizabeth', last_name: 'Lin',
-                            email: 'elizabethylin@gmail.com',
-                            member_role: alumnus_role
-  Member.find_or_create_by! first_name: 'Andy', last_name: 'Qin',
-                            email: 'andrewlqin@gmail.com',
-                            member_role: developer_role
-  Member.find_or_create_by! first_name: 'Jason', last_name: 'Won',
-                            email: 'jason.j.won@gmail.com',
-                            member_role: leader_role
-  Member.find_or_create_by! first_name: 'Daniel', last_name: 'Li',
-                            email: 'danielmxli@gmail.com',
-                            member_role: developer_role
+  roles_hash = {}
+  member_info = YAML::load_file(File.join(__dir__, "members.yml"))
+  member_info.each do |k, v|
```

Comment:
Since we didn't use k anywhere let's just iterate through the values using the hash's [values method](http://ruby-doc.org/core-2.2.2/Hash.html#method-i-values).  We should also use a better var name than k.